react-commenting.js

```diff
@@ -58,14 +58,14 @@ export default App
 class GoodRobot extends Component{
   render(){
     return(
-      // 8.
+      // 8. wrap in a div make output conversation
       <div>
         <h3>Good Robot</h3>
-        {/* 9. */}
+        {/* 9. create an props.output for a robot */}
         <h4>I hear you saying { this.props.userInput }. Is that correct?</h4>
```

Comment:
this is using string interpolation of `this.props` to render the user's input onto the screen